integration/container/mounts_linux_test.go

```diff
@@ -342,15 +342,15 @@ func TestContainerVolumesMountedAsSlave(t *testing.T) {
 
 	// Convert this directory into a shared mount point so that we do
 	// not rely on propagation properties of parent mount.
```

Comment:
Same here?

```suggestion
	// Convert this directory into a private mount point so that we do
	// not rely on propagation properties of parent mount.
```